libs/ui/inputs/src/lib/form-field/form-field.component.ts

```diff
@@ -17,7 +17,9 @@ import { Observable } from 'rxjs'
 export class FormFieldComponent {
   @Input() config: FormFieldConfig
   @Input() set value(v: unknown) {
-    this.formControl.setValue(v)
+    this.formControl.setValue(v, {
+      emitEvent: false,
```

Comment:
I'm curious, what is this for?

Reply:
This is to avoid having an event emitted out of the component when we set its value; otherwise what happens is that the value of the field is changed once the record is loaded, and so the record is "marked as changed", which means that it can be saved although it hasn't received any change yet